doc/manuals/user/pagination.md

```diff
@@ -80,11 +135,15 @@ POST <orion_host>:1026/v1/queryContext?offset=1000&limit=100
 
 ## Ordering results
 
-However, in the case of entities query, the `orderBy` URL parameter can be use to
-specify a different order. In particular, the value of `orderBy` is a comma-separated
-list of attributes and the results are ordered by the first attribute. On ties,
-the results are ordered by the second attribute and so on.
-A "!" before the attribute name means that the order is reversed. For example:
+In the case of entities query, the `orderBy` URL parameter can be use to
+specify an order different from creation time. In particular, the value of
+`orderBy` is a comma-separated list of attributes and the results are ordered
+by the first attribute. On ties, the results are ordered by the second attribute and so on.
+A "!" before the attribute name means that the order is reversed. For example (NGSIv2):
+
```

Comment:
Note that the list can be destroyed anyway in the case of deleting entities already shown in early pages. Thus, pagination may suffer whatever the case :)